.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: python
 python:
   - "3.6"
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y portaudio19-dev libffi-dev libssl-dev
```

Comment:
I think we might still need it, if there is no portaudio wheel for a given platform, I will investigate separately #10